### PR TITLE
Refactored to Polars: Chapter 3

### PR DESCRIPTION
**Pull request by jps215**
Refactor chapter 3

### CODE REVIEW

File: cookbook/Chapter 3 - Which borough has the most noise complaints (or, more selecting data).py [hunk 1]
```diff
@@ -1,93 +1,50 @@
...
 
-
-# Make the graphs a bit prettier, and bigger
-plt.style.use("ggplot")
-plt.rcParams["figure.figsize"] = (15, 5)
-
-# This is necessary to show lots of columns in pandas 0.12.
-# Not necessary in pandas 0.13.
+# To display more columns in Polars
 pd.set_option("display.width", 5000)
```
Comment (ps2016-mt) [inline cookbook/Chapter 3 - Which borough has the most noise complaints (or, more selecting data).py]: Looks good mostly! Just the pd references here throwing it off slightly.

**Review by ps2016-mt — COMMENTED** [review]
Looks good, just a singular comment

File: cookbook/Chapter 3 - Which borough has the most noise complaints (or, more selecting data).py (hunk 1 above)
Comment (yuanjingshi) [reply cookbook/Chapter 3 - Which borough has the most noise complaints (or, more selecting data).py]: It looks it is done by pandas. But the requirement is to do in Polars.

Comment (jps215) [reply cookbook/Chapter 3 - Which borough has the most noise complaints (or, more selecting data).py]: Thanks! I have made the changes

**Review by yuanjingshi — APPROVED** [review]
Looks nice!

**Pull request merged**